### DISCUSSION TITLE
Is Trivy picking up vulnerability CVE-2024-6387

### DISCUSSION THREAD

**Discussion by jodiejones-moj** [aquasecurity/trivy#7090] [Q&A] [answered]
### Question

Hi there, 

The following vulnerability came to our attention yesterday https://blog.qualys.com/vulnerabilities-threat-research/2024/07/01/regresshion-remote-unauthenticated-code-execution-vulnerability-in-openssh-server. Can someone please confirm if Trivy Image Scanning is detecting this vulnerability?

Thanks,
Jodie

### Target

Container Image

### Scanner

Vulnerability

### Output Format

None

### Mode

None

**Comment by knqyf263** [comment, marked as answer]
Yes, Trivy detects it if OpenSSL is installed via OS package manager, such as dnf and apt.
